src/store/__mocks__/devfile.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2018-2020 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+export class DevfileBuilder {
```

Comment:
Do we really need it now? I think Workspace or fake Storage builder will be better.

Reply:
As long as devfile is not only part of a workspace but a thing by itself, then yes, we really need the `DevfileBuilder`. You can also have a look at how it's used.

Reply:
discussed